js/src/behaviors/dismiss.ts

```diff
@@ -0,0 +1,40 @@
+export function initDismiss() {
+	window.addEventListener(
+		'click',
+		({ target }: Event) => {
+			const dismissButton = target instanceof Element && target.closest('[data-dismiss]');
+
+			if (!dismissButton) {
+				return;
+			}
+
+			const dismissTarget = dismissButton.closest('[data-dismissable]');
+
+			if (!dismissTarget) {
+				return;
+			}
+
+			dismissElement(dismissTarget);
+		},
+		{ passive: true }
+	);
+}
+
+async function dismissElement(dismissTarget: Element) {
+	if (dismissTarget.hasAttribute('disappearing')) {
```

Comment:
This should be `data-disappearing`, I think - otherwise browsers will give validation errors about this not being an existent HTML attribute name.

Reply:
Renamed attribute to `data-dismiss-animation`